### PR TITLE
new first and last names in pt_BR

### PR DESCRIPTION
**Pull request by tetri**
I'va added brand new first and last names in proper files

### CODE REVIEW

**Comment by Marak** [comment]
@tetri -

Thank you for your contributions!

We've actually had a few people contribute data to `pt_BR` locale and have expanded the locale.

We can't merged this Pull Request in it's current form since it's conflicting with some of the existing names. It looks like your Pull Request will add more names, so we do want to merge it.

Can you update this Pull Request so that it's not conflicting with `master`? It would be good if we could also review the existing data in `pt_BR` names to ensure accuracy.

**Comment by tetri** [comment]
@Marak 

how to solve the build problems reported by Travis-CI?

File: lib/locales/pt_BR/name/first_name.js [hunk 1]
```diff
@@ -1,171 +1,1211 @@
...
+  "Zilena",
+  "Zilma",
+  "Zilá",
+  "Ziraldo",
+  "Zita",
+  "Zora",
+  "Zoraide",
+  "Zozimo",
+  "Zulmira",
+  "Zyndal"
```
Comment (Marak) [inline lib/locales/pt_BR/name/first_name.js:1207]: Missing comma

**Comment by Marak** [comment]
@tetri -

The tests should pass locally first before code gets pushed to Github. Travis is saying the tests are failing.

The problem is missing comma in the data.

Please review PR and fix should be easy. Thank you!

**Review by tetri — COMMENTED** [review]
fix comma ;)

**Pull request closed without merging**